### DISCUSSION TITLE
How to host a free website on git hub

### DISCUSSION THREAD

**Discussion by Najeeb-Ayo** [community/community#146264] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Pls i have been trying to host on git hub but am having issues loading my CSS file along with the HTML, pls can anyone help me with the steps cause i tried all the step i saw online and the CSS file is not going with the html. Only the HTML file is published. Please anyone who can help am open for discussion

**Comment by cachedcake** [comment, marked as answer]
Hello, @Najeeb-Ayo 
I've looked through your repositories to find the one you're talking about in this discussion and I've spotted your issue, the solution should be the way you're directing the index.html to your assets folder, instead try this:

```html
<link rel="stylesheet" href="styles.css">
```

What I changed:
- Instead of you directing the HTML file to a folder named CSS and then to the styles.css I directly directed it since there is no folder.

> For any further questions you may have use the [GitHub Pages Documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#about-github-pages)

Hope this helps!

**Reply by mecodeatlas** [reply]
Hey, @Najeeb-Ayo!
Did @cakecable's response help you with your question? :)

**Reply by Najeeb-Ayo** [reply]
yes it works thank for the answer am able to add my css file to the html, am new to web development so am grateful seeing that people are still out there to help me through my self learning process
